docs/qtglean.md

```diff
@@ -271,3 +291,5 @@ any call to a Glean API from any language will record to the same Glean instance
 [t-c-g-ffi]: https://searchfox.org/mozilla-central/source/toolkit/components/glean/api/src/ffi
 [glean-bool-set]: https://mozilla.github.io/glean/book/reference/metrics/boolean.html
 [uniffi]: https://mozilla.github.io/uniffi-rs/
+[sarahs-doc]: https://docs.google.com/document/d/1A2O_eACk0P3jDy0g2rRN5UTBOUj-iN-re6tVsRpxtDA/edit?usp=sharing
```

Comment:
This document is marked as confidential, do we still want to expose the link?

Reply:
The link will only work for NDA'd people anyways... It provides useful context for all of this, but I can remove it. Do you think that is problematic?

Reply:
I personally don't mind this, if there are no other concerns I think we can leave it as is